src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -31,11 +31,11 @@ public abstract class ProxyOperationManager
         private readonly int connectionTimeout;
         private readonly string versionCheckPropertyName = "IsVersionCheckRequired";
         private readonly ManualResetEventSlim testHostExited = new ManualResetEventSlim(false);
-        private readonly ManualResetEventSlim testHostLaunchError = new ManualResetEventSlim(false);
+        private readonly ManualResetEventSlim testHostConnectionTimeOut = new ManualResetEventSlim(false);
```

Comment:
nit: timeout is [one word](https://www.google.co.in/search?q=define+timeout&oq=define+timeout&aqs=chrome..69i57j0l5.1672j0j7&sourceid=chrome&ie=UTF-8)